README.md

```diff
@@ -1 +1,111 @@
-# CMS_Public_Use_File
+# CMS Public Use File:<br/>A Machine Learning Case Study
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Dataset](#dataset)
+- [Feature Selection](#feature-selection)
+	- [Human Feature Selection](#human-featur-selection)
+	- [Computer Feature Selection](#computer-feature-selection)
+- [Model Performance](#model-performance)
+- [Conclusion](#conclusion)
+
+
+## Introduction
+The CMS data provides county and statewide Medicare and Medicaid information. There are a few target variables (Actual Per Capita Cost, Average HCC Score, etc.) to choose from and a variety of ‘per capita' explanatory variables (Ambulance Events Per 1000 Beneficiaries, Imaging Events Per 1000 Beneficiaries, etc.) Essentially, we want to see if a human can outperform a machine in predicting the Actual Per Capita Cost target variable.
+
+Here is the competition:
+- The human is only allowed to use graphics to visually pick the 5 most important explanatory variables
+- The computer is allowed to use penalized regression and decision trees to pick the 5 most important explanatory variables
+- Two multiple linear regression models will be created
+  - One with the 5 human chosen variables
+  - One with the 5 computer chosen variables
+- The species with the higher R^2 wins
+
+## Dataset
+A link to download the dataset can be found [here]()
```

Comment:
Best to put a link to the website where you downloaded the date from. Similar to the table of contents syntax, the text in the brackets `[]` is the text that will be displayed, and the text within the parenthesis `()is the location of where the link will go. In this case, it should be a website URL.